package/router/radix/tree_test.go

```diff
@@ -273,6 +274,20 @@ func TestMatch(t *testing.T) {
 	})
 }
 
+func TestMatchUnicode(t *testing.T) {
+	ok(t, &test{
+		inserts: []*insert{
+			{route: "/α"},
+			{route: "/β"},
+		},
+		requests: []*request{
+			{path: "/α", route: "/α"},
```

Comment:
Could we add a request to a capital cased unicode character to see if it will route to the lowercase version?

For example, according to https://en.wikipedia.org/wiki/Greek_alphabet, the following should now work:

- /Δ => /δ